deploy/charts/venafi-kubernetes-agent/values.yaml

```diff
@@ -7,7 +7,7 @@ replicaCount: 1
 
 image:
   # -- Default to Open Source image repository
-  repository: quay.io/jetstack/preflight
+  repository: quay.io/jetstack/venafi-agent
```

Comment:
This is a more appropriate image, but later we should set up mirroring to `registry.venafi.cloud`